src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(a) {
+      return this.result + a;
+    },
+
+    subtract(a) {
+      return this.result - a;
```

Comment:
[CODE STYLE] - Similarly, use `-=` assignment operator to subtract from `this.result`.